src/hotspot/cpu/riscv/c1_LIRGenerator_riscv.cpp

```diff
@@ -772,7 +772,79 @@ void LIRGenerator::do_ArrayCopy(Intrinsic* x) {
 }
 
 void LIRGenerator::do_update_CRC32(Intrinsic* x) {
-  ShouldNotReachHere();
+  assert(UseCRC32Intrinsics, "why are we here?");
```

Comment:
I suppose the performance data is for C2 intrinsic, is there performance data for C1?

Reply:
Measured and provided it in a comment below. I used the `-XX:TieredStopAtLevel=1` flag with the same benchmark on the same machine.